insights/terraform/projects/bfd/athena-queries/partners.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE VIEW "bfd_insights_bfd_prod_partners" AS
+SELECT
+    mdc_bene_id AS benes,
+    CASE "mdc_http_access_request_clientssl_dn"
+        WHEN 'C=US, CN=dpc.prod.client' then 'dpc'
+        WHEN 'C=US,CN=dpc.prod.client' then 'dpc'
+        WHEN 'CN=bluebutton-backend-prod-data-server-client-test' then 'bfd_test'
+        WHEN 'CN=bcda-prod-client' then 'bcda'
+        WHEN 'CN=ab2d-prod-client' then 'ab2d'
+        WHEN 'CN=ab2d-prod-validation-client' then 'ab2d_test'
+        WHEN 'EMAILADDRESS=ryan@adhocteam.us, CN=BlueButton Root CA, OU=BlueButton on FHIR API Root CA, O=Centers for Medicare and Medicaid Services, L=Baltimore, ST=Maryland, C=US' then 'bb2'
```

Comment:
Well that seems.. highly specific. Is that email address likely to change?

Reply:
If I understood correctly, that was what we found in historical log data, character-for-character. Obviously, the historical log entries won't change, but you're absolutely right that it might be worthwhile to handle a slightly more general case, maybe with `CASE ... WHEN field LIKE '%CN=BlueButton Root CA%' ...`. If there's something I can get in relatively short order that doesn't hurt us in speed too much, I'll fix it up.

Reply:
this is what's in the certificate for better or worse and it is a part of the historical data. the unwieldiness of these values is a big reason for having this view. I'd rather not have wildcards since we don't know what they may match unintentionally later on and this is very clear about what maps to what.

Reply:
That makes sense. Then I'll leave it as-is.